docs/docs/caching.md

```diff
@@ -51,7 +51,9 @@ When deploying with Now, follow the instructions in the [Now documentation](http
 
 ---
 
+ <!-- prettier-ignore-start -->
```

Comment:
just checking why was this needed?

Reply:
The changes in this block were not great and @tesserails suggested this change.
https://github.com/gatsbyjs/gatsby/pull/21671#discussion_r383638736